com.xnview.XnConvert.yaml

```
@@ -96,4 +94,10 @@ modules:
       - install -Dm0644 "com.xnview.XnConvert.appdata.xml" -t "${FLATPAK_DEST}/share/appdata/"
       - install -Dm0644 "xnconvert.svg" "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
 cleanup:
+  - /share/doc
```

```suggestion
  - /share/et
```

`/share/doc` does not seem to exist anymore.